templates/templates/docs/shared/_docs.html

```diff
@@ -100,6 +100,49 @@ <h3 class="p-side-navigation__heading">{{ nav_group.navlink_text }}</h3>
     <main class="col-9">
       {{ document.body_html | safe }}
 
+      <div class="p-article-pagination">
```

Comment:
```suggestion
<div class="p-strip">
<hr class="u-no-margin--bottom">
      <div class="p-article-pagination">
```

Reply:
Suggestion looks good to me, thanks! Added with indentation